lib/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -109,6 +111,22 @@ public void deleteVcStoreItemsIfAnyExpired(String userId) {
         }
     }
 
+    public void deleteVcStoreItemsIfAnyInvalid(String userId) {
+        Instant nowPlusSessionTimeout =
```

Comment:
Can put nowPlusSessionTimeout calculation in separate method and call that everywhere.

Reply:
Added shared nowPlusSessionTimeout() method